rpc/backend/utils.go

```diff
@@ -134,7 +134,7 @@ func (b *Backend) processBlock(
 	// set basefee
 	targetOneFeeHistory.BaseFee = blockBaseFee
 	cfg := b.ChainConfig()
-	if cfg.IsLondon(big.NewInt(blockHeight)) {
+	if cfg.IsLondon(big.NewInt(blockHeight + 1)) {
```

Comment:
Can you give some background on where this +1 coming from?

Reply:
it's to [align](https://github.com/ethereum/go-ethereum/blob/v1.10.26/eth/gasprice/feehistory.go#L91) with go-ethereum